zuul.d/molecule.yaml

```diff
@@ -172,6 +172,16 @@
       - ^molecule-requirements.txt
       - ^ci_framework/roles/pkg_build/(?!meta|README).*
       - ^ci/playbooks/molecule.*
+- job:
```

Comment:
I don't see the molecule directory - if we can't run molecule against this role, please check how we did in some other modules, such as here: https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/copy_container/molecule/default

Reply:
thanks, added.